### PR TITLE
[1.16.x] Add function to add items with the same behavior as the pumpkin for enderman

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
This RP adds an function to create an item like carved pumpkin for enderman

This PR is a copy of an [old ](https://github.com/MinecraftForge/MinecraftForge/pull/6759)one which was closed for Forge 1.15.x

### CODE REVIEW

File: src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java [hunk 1]
```diff
@@ -450,4 +451,15 @@ default boolean isRepairable()
...
     }
+
+    /**
+     * Whether this Item can be used to hide player head for enderman.
+     *
+     * @param player The player watching the enderman
+     * @param endermanEntity The enderman that the player look
+     * @return true if this Item can be used.
+     */
+    default boolean isEnderMask(PlayerEntity player, EndermanEntity endermanEntity) {
```
Comment (diesieben07) [inline src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java]: Formatting (braces on newlines)

**Review by diesieben07 — CHANGES_REQUESTED** [review]

File: src/main/java/net/minecraftforge/common/extensions/IForgeItem.java [hunk 2]
```diff
@@ -788,4 +790,17 @@ default void onHorseArmorTick(ItemStack stack, World world, MobEntity horse)
...
+
+    /**
+     * Whether this Item can be used to hide player head for enderman.
+     *
+     * @param stack the ItemStack
+     * @param player The player watching the enderman
+     * @param endermanEntity The enderman that the player look
+     * @return true if this Item can be used to hide player head for enderman
+     */
+    default boolean isEnderMask(ItemStack stack, PlayerEntity player, EndermanEntity endermanEntity) {
```
Comment (diesieben07) [inline src/main/java/net/minecraftforge/common/extensions/IForgeItem.java]: Formatting (braces on newlines)

Comment (Mysterious-Dev) [reply src/main/java/net/minecraftforge/common/extensions/IForgeItem.java]: It's change

File: src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java (hunk 1 above)
Comment (Mysterious-Dev) [reply src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java]: It's change

**Review by diesieben07 — APPROVED** [review]

**Comment by Mysterious-Dev** [comment]
Now what can i do?

**Pull request merged**